pmm-app/tests/QAN/steps/qanActions.js

```diff
@@ -175,7 +175,7 @@ module.exports = {
   },
 
   verifyChangedCount(countBefore, countAfter) {
-    assert.notEqual(countAfter, countBefore, 'Data should be changed');
+    assert.notEqual(countAfter, countBefore, `The value ${countBefore} should not be equal to ${countAfter}`);
```

Comment:
Please add extra Parameter for the type of verification, so in case you are checking results after applying filter, it should say `After applying Filter value ${countBefore} should not be equal to ${countAfter}` 

Another example: when changing Number of Items, by changing items per page, in that case it should says 
`After applying Item Change value ${countBefore} should not be equal to ${countAfter}`